py/emitinlinerv32.c

```diff
@@ -220,6 +245,9 @@ typedef void (*call_i_t)(asm_rv32_t *state, mp_int_t immediate);
 typedef void (*call_r_t)(asm_rv32_t *state, mp_uint_t rd);
 typedef void (*call_n_t)(asm_rv32_t *state);
 
+#define EXT_NIL (0)
+#define EXT_ZBA (1 << (EXTENSION_BIT_ZBA))
```

Comment:
As suggested above, these could be moved to `asmrv32.h` and reused by mpy-cross etc.

Reply:
This should be addressed in the latest version of the PR.